ethereum/core/src/main/java/org/hyperledger/besu/ethereum/chain/GenesisState.java

```diff
@@ -102,13 +102,13 @@ public Block getBlock() {
    * @param target WorldView to write genesis state to
    */
   public void writeStateTo(final MutableWorldState target) {
-    writeAccountsTo(target, genesisAccounts, (Hash) block.getHeader().getBlockHash());
+    writeAccountsTo(target, genesisAccounts, block.getHeader());
   }
 
   private static void writeAccountsTo(
       final MutableWorldState target,
       final List<GenesisAccount> genesisAccounts,
-      final Hash rootHash) {
+      final BlockHeader rootHash) {
```

Comment:
shouldn't we rename this rootHash  to blockHeader ?

Reply:
done.